### DISCUSSION TITLE
UI design selection for Mostro mobile

### DISCUSSION THREAD

**Discussion by Catrya** [MostroP2P/mobile#24] [Polls]
Which design do you prefer for the Mostro mobile client? If you have any suggestions, please comment below.

**1) Tabs with dark background**
![Tabs with dark background](https://github.com/user-attachments/assets/e64521bc-416c-41ce-bb21-7d7a629b0d8b)

**2) Tabs with white background**
![OK Tabs with white background](https://github.com/user-attachments/assets/d954faed-6cce-44e9-a87a-97217de4058c)

**3) Buttons with dark background**
![OK Buttons with dark background](https://github.com/user-attachments/assets/f28b4a9e-2d71-47b4-abdd-2c0eecd636c9)

**4) Buttons with white background**
![OK Buttons with white background](https://github.com/user-attachments/assets/e06e4fa4-c7bb-44ce-84c1-4c3bdd472f54)

**Comment by maxcrowar** [comment]
What about my design?
![Mostro Market Main Sell](https://github.com/user-attachments/assets/d4e65651-11cd-435e-bde8-b1abb83f2297)

**Comment by alltheseas** [comment]
1. please consider not using grey font on white background, grey font on black background + tiny font size. Black and white has the most contrast. This matters especially on small screens. 

2. If you are in the "Buy" tab replace "offering" with "Buy". 

3. 3.9/5 (5) might be confusing for some. I am assuming this is seller rating, and seller review count. Consider using a one, two, three scale instead. Example below with emojis: 
1: 😊 (for instance 4.5/5 or higher)
2: 😬 (e.g. 4.0 - 4.5 ) 
3: ☹️ (below 4.0)

**Reply by grunch** [reply]
Hi @alltheseas thank you for your suggestion, I agree to all of them but not sure about the 3 emoji scale, hope to read you in future discussions about Mostro UI/UX

**Reply by alltheseas** [reply]
The question that inevitably comes up in a 5.0 scale is what does 4.6 mean? How about 3.9? 4.1? 

What I've learned in UI is the problem with arbitrary numeric scales comes to translating "what does this mean". 

Scale doesn't need to be emojis. 

As an example for your research, take edmunds car sale price ratings: great, good, fair. See below:

![image](https://github.com/user-attachments/assets/806f09fd-2c97-4f53-b139-b57ac0b0ee70)

![image](https://github.com/user-attachments/assets/3c4232e1-c5c1-40b2-8414-82e32fac6538)

![image](https://github.com/user-attachments/assets/28599bd8-a11b-4cc6-917b-4194cc1ae980)

![image](https://github.com/user-attachments/assets/d3e0f268-5d76-494b-bbd4-a261e910bca4)

**Reply by grunch** [reply]
Here is how we implemented it on @lnp2pbot
 4.9 is 5 ⭐️
1.5 is 2 ⭐️

we round it up but we still show to the user the rate number, maybe is useful for some users to know that two stars, one with 1.5 and the other one with 2.0 are not exactly the same
  
![image](https://github.com/user-attachments/assets/266d7534-a759-4e1b-a949-dd24d5605c1a)
